Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -670,6 +699,8 @@ open class Kommunicate: NSObject,Localizable, KMPreChatFormViewControllerDelegat
         createConversation(conversation: conversation) { (result) in
             switch result {
             case .success(let conversationId):
+                let teamid = "67476167"
```

Comment:
Make sure to remove this before merging.

Reply:
Sure will remove these line before merging